frontend/packages/pipelines-plugin/src/components/import/pipeline/PacInfoPanel.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import { ClipboardCopy, ClipboardCopyVariant, Text } from '@patternfly/react-core';
+import { FormikValues, useFormikContext } from 'formik';
+import { Trans, useTranslation } from 'react-i18next';
+import { Resources } from '@console/dev-console/src/components/import/import-types';
+import FormSection from '@console/dev-console/src/components/import/section/FormSection';
+import { useActiveNamespace } from '@console/shared/src';
+import './PacSection.scss';
+
+const InfoPanel: React.FC = () => {
+  const { t } = useTranslation();
+  const [namespace] = useActiveNamespace();
+  const { values } = useFormikContext<FormikValues>();
+  const [selectedResource, setSelectedResource] = React.useState<string>();
+
+  React.useEffect(() => {
+    if (values.resources) {
```

Comment:
@Lucifergene shouldnt this value always be there? Do you have a case in mind where it will be null/undefined?

Reply:
I just kept it as a safety check. I have not seen any case where this is null. Removing it